server/agents/project_chat_orchestrator.py

```diff
@@ -380,67 +390,71 @@ def _get_history(project_config: LlamaFarmConfig) -> ChatHistory:
 def _get_client(
     project_config: LlamaFarmConfig,
 ) -> instructor.client.AsyncInstructor | AsyncOpenAI:
-    mode = _determine_instructor_mode(project_config)
+    """Get client for the configured provider using the provider registry.
 
-    if project_config.runtime.provider == Provider.openai:
-        openaiClient = AsyncOpenAI(
-            api_key=project_config.runtime.api_key,
-            base_url=project_config.runtime.base_url,
-        )
-        if project_config.runtime.prompt_format == PromptFormat.structured:
-            return instructor.from_openai(openaiClient, mode=mode)
-        else:
-            return openaiClient
-
-    if project_config.runtime.provider == Provider.ollama:
-        openaiClient = AsyncOpenAI(
-            api_key=project_config.runtime.api_key or settings.ollama_api_key,
-            base_url=project_config.runtime.base_url or f"{settings.ollama_host}/v1",
-        )
-        if project_config.runtime.prompt_format == PromptFormat.structured:
-            return instructor.from_openai(openaiClient, mode=mode)
-        else:
-            return openaiClient
+    DEPRECATED: Use _get_client_for_model instead for multi-model support.
```

Comment:
if this is deprecated, remove the function entirely. no need to keep around dead code